Google.Api.Generator.Tests/ProtoTests/MethodSignatures/MethodSignatures.proto

```diff
@@ -89,6 +89,11 @@ message MessageRequest {
   repeated Msg repeated_required = 4 [(google.api.field_behavior) = REQUIRED];
 }
 
+enum TopLevelEnum {
+	DEFAULT = 0;
```

Comment:
That's a lot of whitespace. Outdent a bit?

Reply:
Tabs :( Done